homeassistant/components/image_processing/facebox.py

```diff
@@ -104,11 +129,22 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     if DATA_FACEBOX not in hass.data:
         hass.data[DATA_FACEBOX] = []
 
+    ip = config[CONF_IP_ADDRESS]
+    port = config[CONF_PORT]
+    username = None
+    if CONF_USERNAME in config:
+        username = config[CONF_USERNAME]
+    password = None
+    if CONF_PASSWORD in config:
+        password = config[CONF_PASSWORD]
+
     entities = []
     for camera in config[CONF_SOURCE]:
         facebox = FaceClassifyEntity(
-            config[CONF_IP_ADDRESS],
-            config[CONF_PORT],
+            ip,
```

Comment:
Please continue the line as far as possible before breaking the line. But keep the first break after the first parenthesis.

This is more in line (😉) with home assistant style.

Reply:
ok